stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/QueryChannelsController.kt

```diff
@@ -117,7 +110,7 @@ public class QueryChannelsController internal constructor(
 
     internal suspend fun handleEvent(event: ChatEvent) {
         if (event is HasChannel) {
-            when (mutableState.eventsHandler.onChannelEvent(event, filter)) {
+            when (mutableState.eventHandler.onChatEvent(event, filter)) {
```

Comment:
This is also super weird - we're using an `onChatEvent` function name result as a way to parse our logic branches

I think it would be better to rename it to something like: `eventHandler.getChatEventResult` or something

Reply:
I fear, we can't rename it currently since it was exposed to some customer

Reply:
Or we can, since we already broke naming of interface :-)

Reply:
Renamed it to `handleChatEvent`